includes/Feed/FeedScheduler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use SkyVerge\WooCommerce\Facebook\Utilities\Heartbeat;
+
+/**
+ * A class responsible for setting up feed generation schedule.
+ */
+class FeedScheduler {
+
+	const FEED_SCHEDULE_ACTION = 'facebook_for_woocommerce_start_feed_generation';
+
+	/**
+	 * Products belonging to what category should be exported.
+	 *
+	 * @var string
+	 */
+	protected $product_category_to_export = array();
```

Comment:
What this? I can't see it used anywhere.

Reply:
Good catch. It was a leftover from WC CSV code.